### PR TITLE
HaxLib Interface Macro

### PR DESCRIPTION
**Pull request by cmester0**
Add interface macro to replace body of hax functions with todo!()

### CODE REVIEW

**Comment by cmester0** [comment]
I have been using this macro in the annotated-core. I would expect `-i` flag to do this functionality, however it seems not to work for removing just the body with `+:`.

**Comment by W95Psp** [comment]
We just chatted about this with @cmester0, let's add a proc-macro crates for macro helpers in the annotated lib core.

**Review by W95Psp — CHANGES_REQUESTED** [review]
As explained in my previous comment, let's add a proc-macro crates for macro helpers in the annotated lib core, and move this there. That makes more sense, and we will be able to name it "backend_primitive" or something like that, which makes sense in the context of the annotated core.

File: hax-lib-macros/src/lib.rs [hunk 1]
```diff
@@ -1000,7 +1022,7 @@ pub fn refinement_type(mut attr: pm::TokenStream, item: pm::TokenStream) -> pm::
             }
 
             #[::hax_lib::exclude]
-            impl #generics ::std::ops::Deref for #ident <#generics_args> {
+            impl #generics ::core::ops::Deref for #ident <#generics_args> {
```
Comment (cmester0) [inline hax-lib-macros/src/lib.rs:1025]: This is still relevant

**Pull request closed without merging**